app/pages/adduser/controller.js

```diff
@@ -1,33 +1,35 @@
-// import axios from 'axios';
+// import axios from 'axios'
+import { getData } from '../../apiProvider';
 import { getContext } from './contextCreator';
-import { identityServer } from '../../config';
+import { organisationApiUrl } from '../../config';
 import { logger } from '../../logger';
 
-export const getAddUserContext = (orgId) => {
-  const endpoint = `${identityServer}/api/v1/Organisations/${orgId}`;
-  logger.info(`api called: [GET] ${endpoint}`);
-  logger.info(`Organisation ${orgId} returned`);
-  return getContext({ data: {} });
+export const getAddUserContext = async ({ organisationId, accessToken }) => {
+  const options = { organisationId };
+  // Need to call this to get org name
+  const orgData = await getData({ endpointLocator: 'getOrgById', options, accessToken });
+  logger.info(`Organisation ${organisationId} returned`);
+  return getContext(orgData);
 };
 
 // TODO: Uncomment when API work is done and remove the implementation above
-// export const getAddUserContext = async (orgId) => {
-//   const endpoint = `${identityServer}/api/v1/Organisations/${orgId}`;
+// export const getAddUserContext = async (organisationId) => {
+//   const endpoint = `${organisationApiUrl}/api/v1/Organisations/${organisationId}`;
 //   logger.info(`api called: [GET] ${endpoint}`);
 //   const response = await axios.get(endpoint);
 
 //   if (response.data) {
-//     logger.info(`Organisation ${orgId} returned`);
+//     logger.info(`Organisation ${organisationId} returned`);
 //     const { data } = response;
 //     return getContext({ data });
 //   }
 
 //   throw new Error('No data returned');
 // };
```

Comment:
Can these comments be removed now. Looks like you've done the work above

Reply:
No because the API work has not been done

Reply:
I just changed bits of this file so that it wasn't a complete mess when it needs updated

Reply:
what do you mean by not done. The only thing that seems to be missing from what you have done above is the error handling. Is this what you mean?

Reply:
the API work is not complete for add user